app/src/js/components/Granules/granule.js

```diff
@@ -119,34 +114,21 @@ class GranuleOverview extends React.Component {
     this.errors = this.errors.bind(this);
     this.selectWorkflow = this.selectWorkflow.bind(this);
     this.getExecuteOptions = this.getExecuteOptions.bind(this);
-    this.state = {};
+    this.state = {
+      workflow: this.props.workflowOptions[0]
+    };
   }
 
   componentDidMount () {
-    const { granuleId } = this.props.match.params;
-    this.cancelInterval = interval(this.queryWorkflows, updateInterval, true);
-
+    this.queryWorkflows();
     if (this.props.skipReloadOnMount) return;
-
-    const immediate = !this.props.granules.map[granuleId];
-    this.reload(immediate);
-  }
-
-  componentWillUnmount () {
-    if (this.cancelInterval) { this.cancelInterval(); }
-  }
-
-  reload (immediate, timeout) {
-    timeout = timeout || updateInterval;
-    const granuleId = this.props.match.params.granuleId;
-    const { dispatch } = this.props;
-    if (this.cancelInterval) { this.cancelInterval(); }
-    this.cancelInterval = interval(() => dispatch(getGranule(granuleId)), timeout, immediate);
+    this.loadGranule();
   }
 
-  fastReload () {
-    // decrease timeout to better see updates
-    this.reload(true, updateInterval / 2);
+  loadGranule () {
+    const { dispatch, match } = this.props;
+    const { granuleId } = match.params;
+    dispatch(getGranule(granuleId));
```

Comment:
Instead of triggering a faster reload, we're just dispatching the action directly on bulk action success

Reply:
that makes sense.